src/github.com/couchbase/sync_gateway/base/bucket.go

```diff
@@ -55,15 +55,56 @@ type BucketSpec struct {
 
 // These are used by CBGT to determine the sharding factor and other properties
 type FeedParams struct {
-	MaxPartitionsPerPIndex int `json:"max_partitions_per_pindex"`
-	NumReplicas            int `json:"num_replicas"`
+	NumShards   int `json:"num_shards"`
+	NumReplicas int `json:"num_replicas"`
```

Comment:
vb number gets stored as uint16 everywhere (including go-couchbase, go-cb), so I think that's the right choice for numShards.  Could probably just use the same for numreplicas.